cluster/gce/container-linux/configure-helper.sh

```diff
@@ -690,7 +710,12 @@ function start-kube-proxy {
   fi
   local container_env=""
   if [[ -n "${ENABLE_CACHE_MUTATION_DETECTOR:-}" ]]; then
-    container_env="env:\n    - name: KUBE_CACHE_MUTATION_DETECTOR\n    value: \"${ENABLE_CACHE_MUTATION_DETECTOR}\""
+    # Daemonset manifest needs a different indent.
+    if [ ${is_daemonset_manifest} = true ]; then
```

Comment:
s/ [ ${is_daemonset_manifest} = true ] / [[ "${is_daemonset_manifest}" = true ]] ?

Reply:
This logic is removed.